src/database/all.sql

```diff
@@ -192,19 +200,77 @@ CREATE TABLE request (
      FOREIGN KEY (client_id) REFERENCES client(client_id),
      FOREIGN KEY (professional_id) REFERENCES client(professional_id),
      FOREIGN KEY (service_id) REFERENCES service(service_id)
-);CREATE TABLE request_subscription (
-	 request_id            MEDIUMINT NOT NULL,
+);  
+  
+------------------------------ 
+-- From: 160_request_subscription.sql 
+CREATE TABLE request_subscription (
+     request_id            MEDIUMINT NOT NULL,
      subscription_id       MEDIUMINT NOT NULL,
      PRIMARY KEY (request_id, subscription_id),
      FOREIGN KEY (request_id) REFERENCES request(request_id),
      FOREIGN KEY (subscription_id) REFERENCES subscription(subscription_id)
-);
-
+);  
+  
+------------------------------ 
+-- From: 170_request_transcation.sql 
 CREATE TABLE request_transaction (
-	 request_id           MEDIUMINT NOT NULL,
+     request_id           MEDIUMINT NOT NULL,
      transaction_id       MEDIUMINT NOT NULL,
      PRIMARY KEY (request_id, transaction_id),
      FOREIGN KEY (request_id) REFERENCES request(request_id),
      FOREIGN KEY (transaction_id) REFERENCES transaction(transaction_id)
-);
+);  
+  
+------------------------------ 
+-- From: 300_insert_billing_type.sql 
+INSERT INTO billing_type (billing_type_name,retired) VALUES ('Out', null);
+
+INSERT INTO billing_type (billing_type_name,retired) VALUES ('In', null);
+
+COMMIT;  
+  
+------------------------------ 
+-- From: 310_insert_subscriptions.sql 
+INSERT INTO subscription (subscription_name, fee) VALUES ('Subscription', 100.50);
+
+INSERT INTO subscription (subscription_name, fee) VALUES ('Single', 0);
+
+COMMIT;  
+  
+------------------------------ 
+-- From: 320_insert_service.sql 
+INSERT INTO service (service_name,  cost, retired) values ('Lawn mowing', 120, null);
+
+INSERT INTO service (service_name,  cost, retired) values ('Weeding', 130, null);
+
+INSERT INTO service (service_name,  cost, retired) values ('Mulching', 150, null);
+
+INSERT INTO service (service_name,  cost, retired) values ('Plumbing Repair', 220, null);
+
+INSERT INTO service (service_name,  cost, retired) values ('Electrical Repair', 250, null);
+
+INSERT INTO service (service_name,  cost, retired) values ('Whitegoods Installation', 110, null);
+
+INSERT INTO service (service_name,  cost, retired) values ('Locks changed', 170, null);
+
+INSERT INTO service (service_name,  cost, retired) values ('Cleaning', 190, null);
+
+commit;  
+  
+------------------------------ 
+-- From: 330_insert_request_status.sql 
```

Comment:
Hey Carl,
Please use these status' instead, this is what Front-End has worked off of. I think at this point it will be easier if we just conform on this.

New
Pending Acceptance
Pending Completion
Complete
Archived